pdf_command_wx.cpp

```diff
@@ -819,26 +822,57 @@ class pdf_illustration : protected html_interpolator, protected pdf_writer_wx
 
         for(auto const& i : pages_)
             {
-            i->pre_render();
+            try
+                {
+                i->pre_render();
+                }
+            catch(const std::runtime_error& e)
+                {
+                std::ostringstream oss;
+                oss
+                    << e.what()
+                    << '\n'
+                    << "Page name: "
```

Comment:
```suggestion
                    << "while preparing to render the page: \""
```
(and also add the closing quote below).